students/laura_denney/lesson06/test_mailroom_part4.py

```diff
@@ -0,0 +1,55 @@
+#-------------------------------------------------#
+# Title: Mail Room Part 4 Unit Testing
+# Dev:   LDenney
+# Date:  November 7, 2018
+# ChangeLog: (Who, When, What)
+#   Laura Denney, 11/7/18, Started MailRoom Part 4
+#   Laura Denney, 11/12/18, Completed work on MailRoom Part 4
+#-------------------------------------------------#
+
+from mailroom_part4 import *
+
+#tests sort_by_amount
+def test_1():
+    assert len(sort_by_amount()) >= 5
+    assert type(sort_by_amount()) is list
+
+#tests the quit function
+def test_2():
+    assert quit() == 'quit'
+
+def test_3():
+    assert get_list() == '\nWe have 5 current donors: John Doe, Laura Denney, Bill Gates, Samuel Jackson, Mr. Bean'
+
+#tests the validation of donation amounts
+def test_4():
+    assert validate_donation(400) == 400.0
+
+def test_4_5():
+    assert validate_donation('four hundred') == 0.0
+
+#tests the function to create a string for report
+def test_7():
+    assert create_string_report() == '\nDonor Name                | Total Given | Num Gifts | Average Gift\n---------------------------------------------------\
+---------------'
+
+def test_7_5():
+    assert create_string_report(sort_by_amount()) > create_string_report()
```

Comment:
this seems like a weird comparison for the test.

I would assert same way you assert in `test_7` but against the new string.